user/how-to-guides/how-to-back-up-restore-and-migrate.md

```diff
@@ -160,6 +160,9 @@ out.
 6. When you are ready, click **Next**. Qubes will proceed to restore from your
 backup. Once the progress bar has completed, you may click **Finish**.
 
+In case that applications are not shown, i.e. "No applications found", open the
+settings of the qube -> select `Applications` -> click `Refresh applications`.
+
 **Note:** When restoring from a dom0 backup, a new directory will be created in
```

Comment:
Hey @MarnixCroes thank you for your suggested contribution!
I would suggest elaborating it further by pointing out that in the case of a restored application qube refreshing the application lists will open the template qube on which it is based (template qube should also be restored in that case, if missing the default qube will be assigned). 
The updated list of the installed software can be seen on the left and adjusted accordingly to your needs.
What do you think?

Reply:
thank you for your review,
I've applied your suggestion

Reply:
merged and corrected  user's needs :)